dist/css/bootstrap.css

```diff
@@ -1442,6 +1442,26 @@ h6,
 .h6 {
   color: #829ca9;
 }
+.display-1 {
+  font-size: 6rem;
```

Comment:
Bootstrap 4 uses rem but we don't, we should map these to pixel size.  Would suggest talking with Julian to come up with sizes.